packages/destination-actions/src/destinations/amplitude/mapUser/generated-types.ts

```diff
@@ -10,7 +10,7 @@ export interface Payload {
    */
   global_user_id?: string
   /**
-   * Amplitude has a default minimum id length of 5 characters for user_id and device_id fields. This field allows the minimum to be overridden to allow shorter id lengths.
+   * Amplitude has a default minimum id length (`min_id_length`) of 5 characters for user_id and device_id fields. This field allows the minimum to be overridden to allow shorter id lengths.
```

Comment:
This change is a result of running `./bin/run generate:types` we can undo it but I think it's fine to keep it.